_tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api.md

```diff
@@ -49,90 +52,289 @@ GET _remotestore/stats/<index_name>
+| `total_download_size.succeeded_bytes` | The total number of bytes of successfully downloaded segment files from remote-backed storage. |
+| `total_download_size.failed_bytes` | The total number of bytes of segment files that failed to download from remote-back storage. |
+| `download_size_in_bytes.last_successful` | The size, in bytes, of the last successful segment file downloaded from remote-backed storage. |
+| `download_size_in_bytes.moving_avg`  | The average size of segement data, in bytes, downloaded in the last 20 downloads. |
+| `download_speed_in_bytes_per_sec.moving_avg` | The average download speed, in bytes per seconds, for the last 20 downloads. |
+
+#### translog
+
+The `translog.upload` object contains the following fields:
 
 |Field	|Description	|
 |:---	|:---	|
-|`refresh_time_lag_in_millis`	|The time (in milliseconds) the remote refresh is behind the local refresh.	|
-|`refresh_lag`	| The number of refreshes by which the remote store is lagging behind the local store.	|
-|`bytes_lag`	| The bytes lag between the remote and local store.	|
-|`backpressure_rejection_count`	| The total number of write rejections made because of remote store backpressure.	|
-|`consecutive_failure_count`	| The number of consecutive remote refresh failures since the last success.	|
-|`total_remote_refresh`	|The total number of remote refreshes.	|
-|`total_uploads_in_bytes`	| The total number of bytes in all uploads to the remote store. |
-|`remote_refresh_size_in_bytes.last_successful`	|The size of data uploaded in the last successful refresh.	|
-|`remote_refresh_size_in_bytes.moving_avg`	|The average size of data (in bytes) uploaded in the last _N_ refreshes. _N_ is defined in `remote_store.segment.pressure.upload_bytes_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).	|
-|`upload_latency_in_bytes_per_sec.moving_avg`	|The average speed of remote store uploads (in bytes per second) for the last _N_ uploads. _N_ is defined in `remote_store.segment.pressure.upload_bytes_per_sec_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).		|
-|`remote_refresh_latency_in_millis.moving_avg`	|The average time taken by a single remote refresh during the last _N_ remote refreshes. _N_ is defined in `remote_store.segment.pressure.upload_time_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).	|
+| `last_successful_upload_timestamp`| The timestamp, in milliseconds, since the last successful translog file uploaded to remore-backed storage. |
```

Comment:
```suggestion
| `last_successful_upload_timestamp`| The timestamp, in milliseconds, since the last translog file successfully uploaded to remote-backed storage. |
```